src/main/java/com/redhat/labs/lodestar/rest/client/GitlabTokenFactory.java

```diff
@@ -0,0 +1,29 @@
+package com.redhat.labs.lodestar.rest.client;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.core.MultivaluedMap;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.rest.client.ext.ClientHeadersFactory;
+import org.jboss.resteasy.specimpl.MultivaluedMapImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@ApplicationScoped
+public class GitlabTokenFactory implements ClientHeadersFactory {
+    public static final String GITLAB_PERSONAL_ACCESS_TOKEN = "GITLAB_PERSONAL_ACCESS_TOKEN";
```

Comment:
I know this copied from activity - I accidentally left this line in the code but it's not needed.

Reply:
removed